packages/app-builder/src/routes/_builder+/lists+/$listId.tsx

```diff
@@ -57,7 +57,7 @@ export default function Lists() {
 
   const columns = useMemo(
     () => [
-      columnHelper.accessor((row) => row.value, {
+      columnHelper.accessor((row) => row, {
```

Comment:
any chance there is another function to call here since the "identity" first argument is kinda useless ?